### PR TITLE
CLDR-15861 Add additional spec text around resource inheritance

### PR DESCRIPTION
**Pull request by sffc**
CLDR-15861

- [x] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by sffc** [comment]
I focused my PR on the new region-priority inheritance. However, it does not contain the following pieces from my [Flexible Vertical Fallback](https://docs.google.com/document/d/1Mp7EUyl-sFh_HZYgyeVwj88vJGpCBIWxzlCwGgLCDwM/edit#) doc:

1. The clean algorithm for language-priority fallback. UTS 35 has the rules for how to handle language-based fallback scattered around the section, with no single step-by-step algorithm, but as far as I can tell, the expected behavior is the same.
2. My doc has a section on fallback with extensions, which I didn't include because it seems rather niche for the CLDR spec. I anticipate that there will be ICU4X-specific documentation about this part.
3. I also did not include the part about the subdivision tag. I could include this if you like.

**Review by macchiati — DISMISSED** [review]

**Review by macchiati — APPROVED** [review]

**Comment by sffc** [comment]
This is ready to merge after another approval

File: docs/ldml/tr35.md
Comment (macchiati) [inline docs/ldml/tr35.md:1480]: Add another note:

Note: Item lookup also requires use of the parentLocales, as described in <a href="#Parent_Locales">Parent Locales</a>.

File: docs/ldml/tr35.md
Comment (macchiati) [inline docs/ldml/tr35.md:1734]: We have never done reattachment of variants before, for normal item lookup. The variant is really more closely attached to the base language than to the region, so I think the "US_someVariant" step should be removed. 

----
Longer term, I think we need a rewrite of inheritance to have clean separation, and treat resource bundle lookup and item lookup in separate sections, and use distinct terminology in each, and always use the same terminology.

**Review by macchiati — CHANGES_REQUESTED** [review]

File: docs/ldml/tr35.md
Comment (macchiati) [reply docs/ldml/tr35.md:1734]: For this to be usable, we have to be precise about exactly which items use region inheritance, and which don't. I'd propose that we say that this applies to all item in rgScope (found in rgScope.xml). Note that some of the attributes that are named "region" take regions or locales, eg

<!ATTLIST hours regions NMTOKENS #REQUIRED >

**Review by macchiati — CHANGES_REQUESTED** [review]

**Comment by macchiati** [comment]
@pedberg-icu , You might look over this PR as well, since it affects inheritance.

**Comment by macchiati** [comment]
This will need to be retargeted at maint/maint-43

File: docs/ldml/tr35.md
Comment (sffc) [reply docs/ldml/tr35.md]: There is already a statement below the example block that says:

> _Both the resource bundle inheritance and the inherited item inheritance use the parentLocale data, where available, instead of simple truncation._

File: docs/ldml/tr35.md
Comment (sffc) [reply docs/ldml/tr35.md]: Thanks for the reference to rgScope. How does this look? I added a link to the existing section on rgScope. Do we need to say anything about attributes named "region" taking regions or locales?

**Review by macchiati — DISMISSED** [review]
Looks ok to me now. @pedberg-icu ?

**Review by pedberg-icu — APPROVED** [review]
Looks good!

**Review by pedberg-icu — APPROVED** [review]

**Pull request merged**
